microservices-orders/docs/index.rst

```diff
@@ -477,8 +491,10 @@ To test your code, save off the project's working solution, copy your version of
 Exercise 5: Stateful operations
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-You can combine current record values with previous record values using aggregations.
-They are stateful operations because they maintain data during processing.
+An aggregation operation takes one input stream or table, and yields a new table by combining multiple input records into a single output record.
+Examples of aggregations are computing `count` or `sum`, because they combine current record values with previous record values.
```

Comment:
```suggestion
Examples of aggregations are computing ``count`` or ``sum``, because they combine current record values with previous record values.
```